### PR TITLE
Issue #9437: update AnnotationLocationCheckTest to use unique Input f…

### LINKED ISSUE #9437: update AnnotationLocationCheckTest to use unique Input file in each test method
AnnotationLocationCheckTest.java file reuse `InputAnnotationLocationIncorrect.java` in few different test method.
Expected: each test method in Test class use it own Input file and do not share it with any other test method.

more details - https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-create-Input-files-for-tests

### LINKED ISSUE #9535: update JavadocParagraphCheckTest to use unique Input file in each test method
JavadocParagraphCheckTest.java file reuse some Input files few time in few different test method.
Expected: each test method in Test class use it own Input file and do not share it with any other test method.

more details - https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-create-Input-files-for-tests

### PR DESCRIPTION
**Pull request by SGanguly1999**
Closes #9437

### CODE REVIEW

**Comment by aryaniiit002** [comment]
try to make it `// OK` and not `// ok`
and also the issue u have mentioned in PR description is already closed
if a separate issue for AnnotationLocationCheckTest is not there u can always make a new issue and work on it. but 1st check if it really not there

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect2.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation;
+
+/* Config:
+ * allowSamelineSingleParameterlessAnnotation = true
+ * allowSamelineParameterizedAnnotation = true
+ * allowSamelineParameterizedAnnotation = true
+ */
+
+@MyAnn_22 @com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation.MyAnn_2 // ok
```
Comment (aryaniiit002) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect2.java:9]: try to make it `// OK` and not `// ok` {OK in caps thats how it should be}
and also the issue u have mentioned in PR description is already closed
if a separate issue for AnnotationLocationCheckTest is not there u can always make a new issue and work on it. but 1st check if it really not there

**Review by aryaniiit002 — COMMENTED** [review]
Please check it

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect2.java (hunk 1 above)
Comment (SGanguly1999) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect2.java:9]: > and also the issue u have mentioned in PR description is already closed

Ohh mistakenly I pasted a different PR. Its now updated

Comment (SGanguly1999) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect2.java:9]: > try to make it `// OK` and not `// ok` {OK in caps thats how it should be}

Ok i will try it. But when I am running the checks in local it is showing we should add `// ok ` not `// OK`

Comment (aryaniiit002) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect2.java:9]: your 1 failing check so to know what's wrong click on details and read it, you will find your problem quickly

**Comment by SGanguly1999** [comment]
@romani @plubdov @strkkk please feel free to review this PR  when you have time.

**Review by romani — CHANGES_REQUESTED** [review]
Please review commits that already merged for the same type of task to see how to update suppression file.

**Comment by SGanguly1999** [comment]
@romani done

**Comment by SGanguly1999** [comment]
@romani please review this when you will have time. Changes requested for the suppression file is done.

**Review by romani — APPROVED** [review]

**Pull request merged**